third-party/realdds/doc/DDS Use cases.md

```diff
@@ -0,0 +1,255 @@
+
+realdds is a library to help control a realsense camera over network using librealsense API.
+It is a work in progress aimed at, ultimately, enabling remote users to do with the camera everything a user can do with a USB connected camera, and more (multiple clients for frames, etc...)
+
+realdds uses DDS topics to send and receive information to/from the camera. See https://github.com/IntelRealSense/librealsense/blob/dds/doc/DDS/DDS%20ICD.md
+
+Users can use realdds stand-alone, or use librealsense with the cmake `BUILD_WITH_DDS` flag on, to use DDS devices seemlessly as any other librealsense device.
+
+# Actors
+
+## Server/Camera
+
+The **server** or **camera** is either a physical device with built in capabilities supporting these use cases, like Intel RealSense D585, or a dedicated software operating some other device, e.g., Intel RealSense D455, giving it the required capabilities.
```

Comment:
Will this branch be merged only to hkr branch later on?

Reply:
No.

It was decided that DDS branch will stay in the public repo.
@Nir-Az is it OK to mention D585 capabilities?